<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
      <item>
       <widget class="QRadioButton" name="first_image_radio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>5</height>
         </size>
        </property>
        <property name="text">
         <string>Image 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="second_image_radio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>5</height>
         </size>
        </property>
        <property name="text">
         <string>Image 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="flow_image_radio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Flow Colorful</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="flow_arrow_radio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Flow Arrowed</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>707</width>
         <height>340</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="imageLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="levels_box">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>2</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="levels">
        <property name="text">
         <string>Levels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="win_box">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="win_size">
        <property name="text">
         <string>Win Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="iterations_box">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="iterations">
        <property name="text">
         <string>Iterations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="polyn_box">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="poly_n">
        <property name="text">
         <string>poly_n</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="polysigma_box">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.300000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.200000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="poly_sigma">
        <property name="text">
         <string>poly_sigma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="gaussian_check">
        <property name="text">
         <string>Gaussian</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCameraCapture"/>
   <addaction name="actionProcessing"/>
   <addaction name="actionLoadImage"/>
   <addaction name="actionSaveImage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadImage">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>LoadImage</string>
   </property>
   <property name="toolTip">
    <string>Load an image from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionProcessing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/process.png</normaloff>:/icons/process.png</iconset>
   </property>
   <property name="text">
    <string>Processing</string>
   </property>
   <property name="toolTip">
    <string>Apply a filter on image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>SaveImage</string>
   </property>
   <property name="toolTip">
    <string>Save the image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/reset.png</normaloff>:/icons/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Undo filter applied </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCameraCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/camera.png</normaloff>:/icons/camera.png</iconset>
   </property>
   <property name="text">
    <string>CameraCapture</string>
   </property>
   <property name="toolTip">
    <string>Capture images from camera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
